Feature: Add tasks for Linux and Windows to create custom Python checks (#408)
DataDog/ansible-datadog: README.md

          - some_data: true
```

##### Custom Python Checks

> To configure a custom check and to pass a Python check to the Playbook, please use the configuration below.  
> This requires the Datadog play / role to be part of a larger Playbook, with the value passed in being the relative filepath 
> to the actual task for [Linux](./tasks/agent-linux.yml) or [Windows](./tasks/agent-win.yml). 
> This is only available for Datadog v6+.  

Reviewer: Would it be helpful to add a reference link for Play and Role (https://docs.ansible.com/ansible/latest/reference_appendices/playbooks_keywords.html#playbook-keywords)?

```suggestion
To pass a Python check to the playbook, use the configuration below. 

This configuration requires the Datadog play and role to be a part of the larger playbook where the value passed in is the relative file path to the actual task for [Linux](./tasks/agent-linux.yml) or [Windows](./tasks/agent-win.yml). 

This is only available for Agent v6+.  
```

> The key should be the name of the file created in the checks directory `checks.d/{{ item }}.py`

Reviewer: ```suggestion
The key should be the name of the file created in the checks directory `checks.d/{{ item }}.py`:
```
